index.html

```diff
@@ -107,14 +107,15 @@ <h1 id="title"><span class="todo">[PROPOSED]</span> Second Screen Working Group
           </tr>
           <tr>
             <th rowspan="1" colspan="1"> Meeting Schedule </th>
-            <td rowspan="1" colspan="1"> Teleconferences: topic-specific calls
-              may be held<br />
-              Face-to-face: we will meet during the W3C's annual Technical
-              Plenary week; other additional F2F meetings may be scheduled (up
-              to 2 per year) <br />
+            <td rowspan="1" colspan="1"> We will meet during the W3C's annual Technical
```

Comment:
@tidoust to clarify if there's an expectation to keep this boilerplate "standard". I note many groups leave the standard text in place even if that does not reflect the realities of the current remote work environment.

I guess the various "may"s are there to give the group flexibility in this regard.

Reply:
No requirement per se. Fine to update if people feel that's needed. I would personally stick to the boilerplate text just to ease re-alignment with the WG charter template next time, but fine updating.

Reply:
Ok, reverted to the boilerplate text :-)

Maybe the charter template could be updated instead...